lib/pure/math.nim

```diff
@@ -153,6 +156,43 @@ func isNaN*(x: SomeFloat): bool {.inline, since: (1,5,1).} =
     when defined(js): fn()
     else: result = c_isnan(x)
 
+func copySign*[T: SomeFloat](x, y: T): T {.inline, since: (1, 5, 1).} =
```

Comment:
cross reference with `sgn` (and, once we have it, `signBit`)

Reply:
It is better to cross reference with `signBit`.